config/settings.d/dhcp_virsh.yml.example

```diff
@@ -2,7 +2,9 @@
 #
 # Configuration file for libvirtd-specific 'virsh' dhcp provider
 #
-# There's currently no configuration options for this provider.
-# Virsh network name is a global parameter that can be set
-# in the main settings.yml file in 'virsh_network' parameter.
-#
+
+# Libvirt network to use
+#:network: default
```

Comment:
Just realised: as you moved the 'network' setting, you'll need to add setting migrations for both virsh_dhcp and virsh_dns modules. There are a few examples in https://github.com/theforeman/smart-proxy/blob/develop/extra/migrations.